12-offer-encoding.md

```diff
@@ -229,6 +229,97 @@ The human-readable prefix for offers is `lno`.
     1. type: 22 (`offer_issuer_id`)
     2. data:
         * [`point`:`id`]
+    1. type: 24 (`offer_recurrence_compulsory`)
+    2. data:
+        * [`recurrence`:`recurrence`]
+    1. type: 25 (`offer_recurrence_optional`)
+    2. data:
+        * [`recurrence`:`recurrence`]
+    1. type: 26 (`offer_recurrence_base`)
+    2. data:
+        * [`recurrence_base`:`recurrence_base`]
+    1. type: 27 (`offer_recurrence_paywindow`)
+    2. data:
+        * [`recurrence_paywindow`:`recurrence_paywindow`]
+    1. type: 29 (`offer_recurrence_limit`)
+    2. data:
+        * [`tu32`:`max_period`]
+
+1. subtype: `recurrence`
+2. data:
+    * [`byte`:`time_unit`]
+    * [`tu32`:`period`]
+
+1. subtype: `recurrence_base`
+2. data:
+    * [`byte`:`proportional_amount`]
+    * [`tu64`:`basetime`]
+
+1. subtype: `recurrence_paywindow`
+2. data:
+    * [`u32`:`seconds_before`]
+    * [`tu32`:`seconds_after`]
```

Comment:
Why is `seconds_before` in u32 (full-length) while `seconds_after` is in truncated u32?

Reply:
Ah, I didn’t know about this, and it looks like this is common with every field. Will look into it more. Thanks!